std_msgs/Bool GPU
---
std_msgs/Bool status
